ref/Lists_of_Geometric_Elements.md

```diff
@@ -6,20 +6,35 @@ This is done with operators that return lists of elements of a particular kind.
 
 #### All points of the construction: `allpoints()`
 
+#### All points incident to an geometric object: `allpoints(‹geo›)`
```

Comment:
Since when is "g" a consonant ;-) thanks again!

Am 24.02.2016 um 10:32 schrieb Stefan Kranich:
>
> In ref/Lists_of_Geometric_Elements.md
> <https://github.com/CindyJS/CindyJS/pull/209#discussion_r53913109>:
>
> > @@ -6,20 +6,35 @@ This is done with operators that return lists of elements of a particular kind.
> >  
> >  #### All points of the construction: `allpoints()`
> >  
> > +#### All points incident to an geometric object: `allpoints(‹geo›)`
>
> @strobelm <https://github.com/strobelm> Sorry to bother you again. It
> should be 'a geometric object', not 'an', in all instances.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CindyJS/CindyJS/pull/209/files#r53913109>.
>

-- 
Michael Strobel, M.Sc.

Technische Universität München -- Zentrum Mathematik
Lehrstuhl für Geometrie und Visualisierung (M10)
Boltzmannstr. 3, 85747 Garching bei München 
https://www-m10.ma.tum.de/bin/view/Lehrstuhl/MichaelStrobel